vllm/model_executor/model_loader/weight_utils.py

```diff
@@ -440,7 +440,13 @@ def safetensors_weights_iterator(
     ):
         with safe_open(st_file, framework="pt") as f:
             for name in f.keys():  # noqa: SIM118
+                if os.environ.get("VLLM_TORCHAX_ENABLED", "0") == "1":
+                    import torchax
+                    torchax.disable_globally()
```

Comment:
I wonder why we do torchax.disable_globally() and then enable_globally

Reply:
yeah me too. I feel this should be a one-liner in the same import torchax block executed at a global level (vllm.env?). 
Then we could just evaluate its static value here instead of calling `os.environ.get` twice.